src/components/FilmDetails.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import cx from 'classnames';
+import { CSSTransitionGroup } from 'react-transition-group';
+
+class FilmDetails extends React.Component {
+
+    constructor() {
+        super();
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    handleClick(e) {
```

Comment:
I would probably create two separate click handlers, one for each action rather than checking classes. It would make the code a tiny bit easier to read